Automattic/wp-calypso
# Checkout: Make sure Themes, Privacy Protection and Google Apps are behind feature flags

Part of #291.

This PR makes sure it's possible to hide the following features (including buttons & links) behind flags:
* [x] Add Privacy Protection
* [x] Add Google Apps
* [x] Add Theme

It also renames `premium-themes` flag to `upgrades/premium-themes` to keep everything consistent.

### Testing
### Testing
1. Disable the following flags in the config file:
 * `upgrades/checkout`
 * `upgrades/premium-themes`
2. Execute `make run` and go to http://calypso.localhost:3000/.
3. Go to My Sites.
4. Select a Site.
5. Go to Themes.
6. You should see only free themes.
7. Go to Domain Management section.
8. Go to Email tab.
9. There should be no Add Google Apps button.
10. Go back to Domains list.
11. Select domain with Google App purchase, and go to accounts list.
12. It should not be possible to add another account (no button).
13. Go back to Domains list.
14. Select a domain without Privacy Protection.
14. Then go to Privacy Protection page.
15. It should not be possible to Add Privacy Protection (no button)

Now enable flags listed in (1) and repeat all other steps, you should see all action buttons and links and premium themes :)
